.gitpod.yml

```diff
@@ -0,0 +1,3 @@
+tasks:
+  - init: pyenv local 3.7.2 && pip install -e ".[dev]"
+    command: python setup.py develop
```

Comment:
No need to run `python setup.py develop`, as it is already installed by the previous line.